docs/unified-test-documentation/dasharo-security/206-secure-boot.md

```diff
@@ -131,3 +131,171 @@ enabled:
 ```powershell
 True
 ```
+
+## SBO003.001 Attempt to boot file with the correct key from Shel (firmware)
+
+**Test description**
+
+This test verifies that Secure Boot allows booting a signed file with a correct
+key.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+1. `USB device`
+
+**Test setup**
+
+1. Proceed with the
+    [Generic test setup: firmware](../../generic-test-setup/#firmware).
+
+**Test steps**
+
+1. Download the file and the certificate from the
+    [cloud](https://cloud.3mdeb.com/index.php/apps/files/?dir=/projects/3mdeb/Dasharo/TAT/SecureBoot_testing_files)
+    or prepare it yourself.
+1. Place the certificate and the file on the `USB device`.
+1. Plug the `USB device` into DUT.
+1. Power on the DUT.
+1. While the DUT is booting, hold the `BIOS_SETUP_KEY` to enter the UEFI Setup
+    Menu.
+1. Enter the `Device Manager` menu using the arrow keys and Enter.
+1. Enter the `Secure Boot Configuration` submenu.
+1. Set the `Secure Boot Mode` field to `Custom Mode`.
+1. Select options in the given order: `Custom Secure Boot Options` ->
+    `DB Options` -> `Enroll Signature` -> `Enroll Signature Using File`
+1. Select the certificate from the `USB device`.
+1. Select the `Commit Changes and Exit` option.
+1. Press `ESC` until the setup menu.
+1. Select the `Reset` option.
+1. While the DUT is booting, hold the `BOOT_MENU_KEY` to enter the boot menu.
+1. Select the `UEFI Shell` option using the arrow keys and press `Enter`.
+1. In the shell open the `USB device` by executing the following command:
+
+    ```bash
+    FS0:
+    ```
+
+    > One of the filesystems in the FS list will be the USB device - typically
+    > `FS0:`
+
+1. Boot the previously prepared file by typing its full name:
+
+    ```bash
+    hello-valid-keys.efi
+    ```
+
+**Expected result**
```

Comment:
The expected result section should be rewritten.

If there are two conditions for passing the test, we should describe them in points, i. e:

```
1. File boots correctly (no information: `Command Error Status: Access Denied` on the output).
2. Output of the command shows file content.

Example output:
```

Reply:
updated